### PR TITLE
Add Mono 4.2.3.4

### PR DESCRIPTION
**Pull request by dominikhahn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Supported tags and `Dockerfile` links
 
-- [`latest` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.2.30/Dockerfile)
-- [`onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.2.30/onbuild/Dockerfile)
+- [`latest` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.3.4/Dockerfile)
+- [`onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.3.4/onbuild/Dockerfile)
+- [`4.2.3.4` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.3.4/Dockerfile)
+- [`4.2.3.4-onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.3.4/onbuild/Dockerfile)- [`4.2.3.4` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.3.4/Dockerfile)
```
Comment (akoeplinger) [inline README.md]: This is all in one line, you need to add a newline before the `4.2.3.4` entry

Comment (akoeplinger) [inline README.md]: wait, it's just duplicated from above, so remove it here :smile:

Comment (dominikhahn) [inline README.md]: Sorry about that :-)

**Comment by akoeplinger** [comment]
LGTM, thanks!

**Comment by akoeplinger** [comment]
PR for docker library: https://github.com/docker-library/official-images/pull/1549

**Pull request merged**